src/biokbase/narrative/jobs/job.py

```diff
@@ -199,7 +226,7 @@ def __getattr__(self, name):
                 # and need the state refresh.
```

Comment:
Comment about the methods `query_ee2_state` and `query_ee2_states` -- I don't find it very intuitive that they're part of the job object, given that one deals with multiple jobs. I think it'd be better to have them in the Job Manager, along with the other ee2 job status query that fetches jobs by workspace ID.

Reply:
I actually kind of agree with what you're arguing here. But I took a look at changing it and it looks like `Job` has invocations like `cls.query_ee2_states` but `Job` doesn't know about `JobManager` so I think we should just keep it as is for now

Reply:
I guess you could also move the invoking method, `from_job_ids`, to `JobManager`, but I kind of like it (because I wrote it hehe ... a class factory method). But I'd argue not in scope of this ticket!

Reply:
But the singular `query_ee2_state` is also invoked in a `Job` instance method
I guess I have a weird habit of putting meta-instance methods all as class methods into that class ... Like I might merge `JobManager` into `Job` if it weren't so ginormous. I wonder if that's an anti-pattern